### DISCUSSION TITLE
Missing file

### DISCUSSION THREAD

**Discussion by vovoRRR** [mikumifa/biliTickerBuy#99] [提问专区]
打开后显示此异常，请问这个文件是必须的吗？是我的电脑屏蔽了还是本来就没有？
Could not create share link. Missing file: C:\Users\XX\Desktop\main\_internal\gradio\frpc_windows_amd64_v0.2.

Please check your internet connection. This can happen if your antivirus software blocks the download of this file. You can install manually by following these steps:

1. Download this file: https://cdn-media.huggingface.co/frpc-gradio-0.2/frpc_windows_amd64.exe
2. Rename the downloaded file to: frpc_windows_amd64_v0.2
3. Move the file to this location: C:\Users\XX\Desktop\main\_internal\gradio

**Comment by mikumifa** [comment]
不用管他，他的作用能让你在手机上也能连上
不过要开梯子才能下载
不下载也能正常使用
